android/app/src/main/java/com/cBreez/client/DismissNotification.kt

```diff
@@ -0,0 +1,13 @@
+package com.cBreez.client
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationManagerCompat
+import com.cBreez.client.Constants.NOTIFICATION_ID_PAYMENT_FAILED
+
+class DismissNotification : BroadcastReceiver() {
```

Comment:
Do we use this or can be removed?

Reply:
Removed as part of [Remove payment hash poller & everything related to failed payment not…](https://github.com/breez/c-breez/pull/748/commits/fc1bb41f68315a4d63fbac3955c802d002d00a73)